sale_order_type/data/default_type.xml

```diff
@@ -1,7 +1,7 @@
 <odoo noupdate="1">
     <record id="normal_sale_type" model="sale.order.type">
         <field name="name">Normal Order</field>
-        <field name="company_id" eval="False" />
+        <field name="company_id" ref="base.main_company" />
```

Comment:
@iTecan This is not correct.

As there is this record rule : https://github.com/OCA/sale-workflow/blob/16.0/sale_order_type/security/security.xml#L7

So, if no company is defined, everybody can see it and it will be available of course in non-multicompany environments